.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.9'
+
+networks:
+  local_network:
+    name: local_network
+
+services:
+  redis:
+    build: ./redis
```

Comment:
Since you're not doing anything special in the docker file i think you can just use `image: redis:7.2.3-alpine` and remove the `build:`.

Reply:
done